### PR TITLE
Update make with other options

### PR DESCRIPTION
**Pull request by epcim**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -373,3 +373,11 @@
 [submodule "formulas/stackstorm"]
 	path = formulas/stackstorm
 	url = https://github.com/salt-formulas/salt-formula-stackstorm
+[submodule "apache"]
```
Comment (fpytloun) [inline .gitmodules]: formulas/apache but it's aodh formula

File: Makefile [hunk 2]
```diff
@@ -24,8 +26,67 @@ release:
...
+FORKED_FORMULAS_DIR=fformulas
+FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+
+scripts_prerequisites:
+	pip3 install -r scripts/requirements.txt
+
+list: scripts_prerequisites
+	@echo $(FORMULAS)
+
+update_forks: scripts_prerequisites
```
Comment (fpytloun) [inline Makefile]: You don't want to clone your forks into different directory, that consumes disk space and doesn't make any sense. You want to use remote pointing to your fork instead.

File: Makefile [hunk 3]
```diff
@@ -24,8 +26,67 @@ release:
...
 	make -C doc html
 
 pdf:
 	make -C doc latexpdf
+
+FORKED_FORMULAS_DIR=fformulas
+FORMULAS=`python3 -c 'import sys; sys.path.append("scripts");from update_mrconfig import *; print(*get_org_repos(make_github_agent(), "salt-formulas"), sep="\n")'| egrep 'salt-formula-' | sed 's/salt-formula-//'`
+
+scripts_prerequisites:
+	pip3 install -r scripts/requirements.txt
```
Comment (fpytloun) [inline Makefile]: No way, you don't want to let makefile silently mess your system when running eg. `make list`

File: Makefile [hunk 4]
```diff
@@ -24,8 +26,67 @@ release:
...
+
+update_forks: scripts_prerequisites
+	@mkdir -p $(FORKED_FORMULAS_DIR)
+	@for FORMULA in $(FORMULAS) ; do\
+     echo "## Forking: $$FORMULA";\
+     test -e $(FORKED_FORMULAS_DIR)/$$FORMULA || git clone https://github.com/salt-formulas/salt-formula-$$FORMULA.git $(FORKED_FORMULAS_DIR)/$$FORMULA;\
+   done;
+
+
+GERRIT_REMOTE_URI=gerrit.mcp.mirantis.net:29418/salt-formulas
```
Comment (fpytloun) [inline Makefile:56]: No way, there's no community gerrit and this is community project

**Review by fpytloun — CHANGES_REQUESTED** [review]

File: Makefile (hunk 4 above)
Comment (fpytloun) [reply Makefile:56]: *closing both eyes*

File: .gitmodules (hunk 1 above)
Comment (epcim) [reply .gitmodules]: will fix or remove

**Pull request merged**